### ISSUE TITLE
 [dupeguru-me] WAV-support

### ISSUE THREAD

**Issue by DJCrashdummy** [hsoft/dupeguru#480]
similar to #476 it seems WAV-files are also completely ignored, no matter if you select _content_ or _filename_ as scan-type (for sure _tags_ won't provide results).

...this leads to the curious situation that if you compare WAV-files no duplicates are shown in the _Music Mode_ (aka `dupeguru-me`), but after changing to _Standard Mode_ the duplicates of these music files are listed. :confused: (the same applies to OPUS-files.)

---
btw: please list somewhere the supported file-types/-extensions of the _Music_ and _Picture Mode_; maybe at the about- or help-menu... or at least at the homepage or github.

**Comment by hsoft** [comment]
Yes, you're right, this documentation is lacking since the 3 editions merge.

Raw information is at https://github.com/hsoft/hsaudiotag/blob/master/hsaudiotag/auto.py#L15

Files like OPUS and WAV aren't supported because hsaudiotag doesn't support them and we can't have their bitrate and duration.

Documentation pull request would be welcomed.

**Comment by DJCrashdummy** [comment]
thanks for the link! - what are your priorities for the documentation to be placed?

well... WAV-support won't add that much benefits, but would be nice anyway.

**Comment by hsoft** [comment]
I've pretty much [stopped working on dupeGuru](https://github.com/hsoft/dupeguru#slowed-development). I welcome outside contribution and will review pull request but I'd really like to get rid of the maintainership burden of this app.
